sources/packages/api/src/cra-integration/cra-integration.service.ts

```diff
@@ -167,38 +163,30 @@ export class CRAIntegrationService {
     }
   }
 
-  private createFooter(
-    code: TransactionCodes,
-    processDate: Date,
-    sequence: number,
-    recordCount: number,
-  ): CRAFileFooter {
-    const footer = new CRAFileFooter();
-    footer.transactionCode = code;
-    footer.processDate = processDate;
-    footer.programAreaCode = this.craConfig.programAreaCode;
-    footer.environmentCode = this.craConfig.environmentCode;
-    footer.sequence = sequence;
-    footer.recordCount = recordCount + 2; // Must be the number of records + header + footer.
-    return footer;
-  }
-
   /**
    * Expected file name of the CRA request file.
    * @param sequence file sequence number.
    * @returns Full file path of the file to be saved on the SFTP.
    */
-  createRequestFileName(sequence: number): string {
+  createRequestFileName(sequence: number): {
+    fileName: string;
+    filePath: string;
+  } {
     const sequenceFile = sequence.toString().padStart(5, "0");
-    return `${this.craConfig.ftpRequestFolder}\\CCRA_REQUEST_${this.craConfig.environmentCode}${sequenceFile}.DAT`;
+    const fileName = `CCRA_REQUEST_${this.craConfig.environmentCode}${sequenceFile}.DAT`;
+    const filePath = `${this.craConfig.ftpRequestFolder}\\${fileName}`;
+    return {
+      fileName,
+      filePath,
+    };
   }
 
   /**
-   * Downloads all files contents present on CRA response file that follows
-   * the regex pattern '/CCRA_RESPONSE_[\w]*\.txt/i'.
-   * @returns File records for all response files present on sFTP.
+   * Get the list of all response files waiting to be downloaded from the
+   * sFTP filtering by the the regex pattern '/CCRA_RESPONSE_[\w]*\.txt/i'.
```

Comment:
sFTP? is it Secure File Transfer Protocol?,  then caps `S`

Reply:
Yes, SFTP is encrypted and does not transfer any data in cleartext. This encryption is the additional layer of security that you don’t get with FTP.

Reply:
ya, I believe it should be `SFTP` instead of `sFTP` right?

Reply:
This is not part of this PR and this will affect more files completed outside of the PR. I can change it more to the end of the review to avoid disturbing the review and including more changed files that are not related to this change 😉 
When I mentioned "it is not part of this PR" I meant that the naming was given way before our recent convention 😄